### DISCUSSION TITLE
探討AI人工智慧 對人類生活的影響

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#195] [General]
AI人工智慧是近幾年開始崛起的科技業藍海，有別於一般大眾認為AI就等同於機器人，其實人工智慧包含的更廣泛且大多運用在幕後，如搜尋引擎、語音辨識。

http://shuj.shu.edu.tw/blog/2017/10/18/%E6%8E%A2%E8%A8%8Eai%E4%BA%BA%E5%B7%A5%E6%99%BA%E6%85%A7-%E5%B0%8D%E4%BA%BA%E9%A1%9E%E7%94%9F%E6%B4%BB%E7%9A%84%E5%BD%B1%E9%9F%BF/

**Comment by Shiba010** [comment]
AI的應用層面真的很廣，這已經是現代世界的潮流。

**Comment by ghost** [comment]
未來感覺AI會越來越多 越貼近我們的生活

**Comment by lx3211** [comment]
現在有chatGPT套件，甚至AI已經可以跟我們對話了，還可以幫我們寫程式、文章...

**Comment by BingRong1217** [comment]
AI對我們的生活有極大的影響

**Comment by skyling0299** [comment]
利用AI能夠更好的幫助人類社會，但同時過度相信AI也可能造成不良後果

**Comment by Ben6uo** [comment]
AI 技術可以幫助提高生產力和效率，使商業運營更加高效

**Comment by limitx0** [comment]
AI很值得研究，也對人類生活影響重大

**Comment by larry40305** [comment]
AI會越來越融入生活之中，同時也會衍生各種利用AI的犯罪行為

**Comment by boson13579** [comment]
適當的運用AI，能為我們生活解決很多不必要的麻煩。
